### ISSUE TITLE
Error with "nanopolish call-methylation"  caused by vbz-compressed fast5 and HDF5

### ISSUE THREAD

**Issue by valerieT1998** [jts/nanopolish#1063]
Hello,

I'm opening a new issue because I am having a hard time debugging this on my own. I am having the exact same issue as  BigNianNGS reported in his issue here https://github.com/nanoporetech/vbz_compression/issues/5 . I've read your suggestions and wanted to try `export HDF5_PLUGIN_PATH=/usr/local/hdf5/lib/plugin` but can't seem to find the lib folder in my `nanopolish/hdf5-1.8.14` directory. Am I looking at the wrong place ?
Also, since I am running this tool in a virtual environment I also tried looking in my virtual env `lib` folder but only found these files 
<img width="280" alt="Screen Shot 2023-02-22 at 10 47 41 AM" src="https://user-images.githubusercontent.com/45580098/220677249-53ecf8a8-fada-4d20-84fb-00c09e960d87.png">

Thanks for your help

**Comment by jts** [comment]
Hi,

You need to first install the plugin from the linked repository by following these instructions:

https://github.com/nanoporetech/vbz_compression#installation

Once you installed that plugin you should set the path using the `export` command.

I hope that helps,
Jared

**Comment by valerieT1998** [comment]
HI,

I tried installing the plugin from source code ( unable with wheel at the moment it's not available on the cluster I'm working with or any other format available). I exported the path to the vbz_compression-1.0.2/vbz_plugin directory in my script` export HDF5_PLUGIN_PATH=/PATH_TO/Nanopolish/vbz_compression-1.0.2/vbz_plugin/:$HDF_PLUGIN_PATH` but still get an error. 
<img width="1440" alt="Screen Shot 2023-02-23 at 2 18 25 PM" src="https://user-images.githubusercontent.com/45580098/221008398-4ce19ef8-f8eb-4fe6-abc0-ac9af65fe4a3.png">
Do I need to compile the vbz_compression ( ex: make ..?) I only uncompressed the tar file, not sure if I need to do something else. 

Thanks
